resources/assets/enderio/lang/zh_CN.lang

```diff
@@ -1330,16 +1364,23 @@ enderio.top.redstone.true=开
 enderio.top.redstone.false=关
 enderio.top.tank.content=%s/%s mB
 enderio.top.tank.content.empty=空
-enderio.top.tank.header.input=输入储罐：%s
-enderio.top.tank.header.output=输出储罐：%s
+enderio.top.tank.header.input=输入槽：%s
+enderio.top.tank.header.output=输出槽：%s
 enderio.top.tank.header.storage=%s
+enderio.top.rf.header.maxin=最大输入:
+enderio.top.rf.header.maxout=最大输出:
+enderio.top.rf.header.avg=I/O:
+enderio.top.rf.value=%s RF/t
+enderio.top.paint.header=涂覆:
+enderio.top.owner.header=猪人:
```

Comment:
Never trust machine translator... for minecraft terminology please refer to minecraft wiki